### DISCUSSION TITLE
Introducing myself: I'd love to join the Devicons organization

### DISCUSSION THREAD

**Discussion by Panquesito7** [devicons/devicon#517] [General]
Hi everyone. Hope you're doing fine today.
In this topic, I will introduce myself to the community. 🙂

**Please check out my [Developer story](https://stackoverflow.com/story/Panquesito7) and my [Liberapay](https://liberapay.com/Panquesito7/)
profile to know the work I do, what do I do, etc..**

As I said in the title, I'd love to join the Devicons organization.

If you have already read the work I do, share your
opinion and tell me what do you think about this.

Thank you! 🙂

**Comment by amacado** [comment]
You have my approval to join @devicons/supporter! (see https://github.com/devicons/devicon/pull/516) I would be happy to have you in the team! @Thomas-Boi?

**Comment by Thomas-Boi** [comment]
You have a lot of experience @Panquesito7!  I'm glad that you decided to stick around after opening your first PR here and I would love to have you join Devicon 😄.

**Comment by Thomas-Boi** [comment]
Hey @Panquesito7,

Congrats on getting accepted! 

I just want to say that if you need any help explaining the GitHub Actions and how we merge the PRs, just let me know 👍. I'll try to be around but I'm currently still in school and things are getting a bit busy for me 😅.

**Reply by Panquesito7** [reply]
Thank you @Thomas-Boi! I'll reach around to you if I need any help. You can also contact me via [Discord](https://discordapp.com/users/759196962595143691) if you like. Thanks! 😃
